skripchenkoveta/hometask-5/4-5.sql

```diff
@@ -0,0 +1 @@
+SELECT name, COUNT(film_id) FROM film JOIN language ON language.language_id = film.language_id GROUP BY name;
```

Comment:
works as should work, but please be more specific when referring to fields, because when I read this statement and reach `GROUP BY name` section I don't understand whether it's `film.name` or `language.name`. I know `film` does not have a `name` field - it has `title` field, but I don't want to make this research, I want to read the statement and have no questions :)

`SELECT language.name, COUNT(*) FROM film JOIN language ON language.language_id = film.language_id GROUP BY language.name;`
or
`SELECT lang.name, COUNT(*) FROM film JOIN language as lang ON lang.language_id = film.language_id GROUP BY lang.name;`